packages/form-js-editor/src/features/render-injection/slot-fill/SlotFillRoot.js

```diff
@@ -5,16 +5,21 @@ import { useMemo, useState } from 'preact/hooks';
 export default (props) => {
   const [ fills, setFills ] = useState([]);
 
+  const {
+    onSetFill,
+    onRemoveFill
```

Comment:
Actually this would lead to changing the context every render pass which means every dependent would constantly re-render. I'm actually thinking towards adding a linting rule that prevents us from setting empty objects as default to props.

Reply:
I've gone ahead with a local noop.